packages/venia-ui/lib/components/LegacyMiniCart/editItem.js

```diff
@@ -55,51 +56,10 @@ export const PRODUCT_DETAILS = gql`
     query productDetailBySku($sku: String) {
         products(filter: { sku: { eq: $sku } }) {
             items {
-                __typename
                 id
-                name
-                sku
-                url_key
-                ... on ConfigurableProduct {
-                    configurable_options {
-                        attribute_code
-                        attribute_id
-                        id
-                        label
-                        values {
-                            default_label
-                            label
-                            store_label
-                            use_default_value
-                            value_index
-                            swatch_data {
-                                ... on ImageSwatchData {
-                                    thumbnail
-                                }
-                                value
-                            }
-                        }
-                    }
-                    variants {
-                        attributes {
-                            code
-                            value_index
-                        }
-                        product {
-                            id
-                            media_gallery_entries {
-                                id
-                                disabled
-                                file
-                                label
-                                position
-                            }
-                            sku
-                            stock_status
-                        }
-                    }
-                }
+                ...ProductDetailsFragment
```

Comment:
Same comment here. Unfortunately you cannot just re-use the fragment here and the content needed by the `RootComponent/Product` and the content needed by this component are different. Re-using the fragment will result in over-fetching of data. 

Please revert this change, or create an `editFormFragment`.